presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -83,6 +83,18 @@ public void testDropTable()
     }
 
     @Test
+    public void testInsertIntoTable()
+            throws Exception
+    {
+        execute("CREATE TABLE test_insert AS SELECT 123 x, 'insert' y limit 0");
+        execute("INSERT INTO test_insert values (123, 'insert')");
```

Comment:
I'm not too familiar with these tests, but it seems like "execute" runs the query on mysql